src/styles/index.ts

```diff
@@ -1673,6 +1673,12 @@ const styles = (theme: ThemeColors) =>
             justifyContent: 'center',
         },
 
+        customEmoji: {
+            alignItems: 'center',
+            justifyContent: 'center',
+            verticalAlign: 'bottom',
```

Comment:
```suggestion
            verticalAlign: 'bottom',
            ...(getPlatform() === CONST.PLATFORM.IOS || getPlatform() === CONST.PLATFORM.ANDROID) ? {marginBottom: -variables.iconSizeNormal/4} : {},
```

If we use this we can remove the `positionFix` completely in `CustomEmojiRenderer` and `FloatingActionButtonAndPopover`.